scalyr_agent/monitor_utils/k8s.py

```diff
@@ -825,43 +850,44 @@ def _build_node_filter( self, k8s, namespaces_to_ignore ):
 
         return result
 
-    def configure( self, new_config ):
+    def configure(self, new_cache_config):
```

Comment:
adding "cache" to the variable name avoids confusion with configuration/k8s_config objects